static/app/components/dropdownMenuV2.tsx

```diff
@@ -74,11 +74,12 @@ function Menu({
   ...props
 }: Props) {
   const state = useTreeState<MenuItemProps>({...props, selectionMode: 'single'});
-  const stateCollection = [...state.collection];
+  const stateCollection = useMemo(() => [...state.collection], [state.collection]);
```

Comment:
🤔 Why do we need to memoize this?

Reply:
Because it's used as a dependency further below

Reply:
Otherwise I get this message:
<img width="936" alt="Screen Shot 2022-04-20 at 2 21 03 PM" src="https://user-images.githubusercontent.com/44172267/164325022-e0c82406-460b-4082-9aee-711769f61203.png">

Reply:
I think you don't need to make a copy of the array do you? can you just pass it as a dependency and use it as `state.collection` instead of having the interim value?